### ISSUE TITLE
Requesting a new npm release (for sepBy & sepBy1)

### ISSUE THREAD

**Issue by beaugunderson** [jneen/parsimmon#57]
Went to use `sepBy` and noticed it wasn't in the current npm release. :)

**Comment by beaugunderson** [comment]
Oh, maybe it's not there for a reason; I installed & built the latest code and my parser hangs indefinitely when I use `sepBy`.

**Comment by anko** [comment]
Latest parsimmon on npm is 0.7.0:

```
$ npm view parsimmon .version
0.7.0
```

Latest tag pushed to the repo is 0.6.0:

```
$ git ls-remote --tags https://github.com/jneen/parsimmon.git | tail -n1
7c4ee36230a0540a08231b7e61d6edf951fa72fc	refs/tags/v0.6.0
```

@beaugunderson Could you confirm what version or commits you're using in each of those cases? A small example of the failing code would help too.

@jneen Could you push the `v0.7.0` tag? I assume it's on 458e7cc; `src/parsimmon.js` matches what's on npm.

**Comment by beaugunderson** [comment]
0.7.0 does not contain `sepBy` and `sepBy1`, was my point. :)

I'll try to make a simple failing test case for `sepBy`.

**Comment by beaugunderson** [comment]
The version I built and tried myself was 666333e, FYI.

**Comment by yanatan16** [comment]
:+1:

**Comment by wavebeem** [comment]
I just released v0.7.1 to npm and installed it locally. sepBy and sepBy1 appear to be in that release now. Please let me know if the issue is still present.
